doctordroid/README.md

```diff
@@ -0,0 +1,23 @@
+# Doctor Droid
+
+## [](https://github.com/DataDog/integrations-extras/tree/master/doctordroid#overview)Overview
+
+Doctor Droid is an observability analytics platform. This integration allows Datadog users to analyse their alerts history to decide better thresholds for alerts, identify alerts coverage of their services and helps the user fetch metrics from upstream and downstream services to enrich their alerts for faster diagnosis.
```

Comment:
```suggestion
Doctor Droid is an observability analytics platform. This integration allows Datadog users to analyze their alerts history to decide better thresholds for alerts and identify alerting coverage for their services. Users can also fetch metrics from upstream and downstream services to enrich their alerts for faster diagnosis.
```

Reply:
Can you also elaborate a bit on what kind of metrics are provided and how they're valuable to users?